WTWSMS/common/buildings/WtWSMS_historical_temple.txt

```diff
@@ -0,0 +1,183 @@
+temple = {
+
+	tp_hagiasophia = {  # Hagia Sophia
+		potential = {
+			FROMFROM = { title = b_hagiasophia }
+			FROM = { had_global_flag = { flag = hagia_sophia_built days = 0 } }
+		}
```

Comment:
Given that this one is done through decision, it shouldn't be possible to build it directly or at the same time through other means. `always = no` would perhaps be appropriate.